src/components/Cards/CartCard.jsx

```diff
@@ -0,0 +1,88 @@
+const CartCard = () => {
+    return(
+<section class="cart-grid">            
+            <div class="card horizontal-large card-box-shadow cart-card m-t20">
```

Comment:
Try to reuse like map CartCard while passing data form Product page

Reply:
Added this page, to avoid the netlify issue. Still Working on this page.